hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -355,6 +355,10 @@
         clk_i: "io_div4",
         clk_slow_i: "aon",
         clk_lc_i: "io_div4",
+        clk_cpu_i: {
+          clock: "main",
+          group: "infra",
```

Comment:
Not sure if these are the clock and reset groups that should be used. I'm not clear on infra vs. secure vs. powerup as choices here.